spyder/app/mainwindow.py

```diff
@@ -2375,40 +2376,75 @@ def report_issue(self, traceback=""):
         revision = ''
         if versions['revision']:
             revision = versions['revision']
+        reminder_message = (
+            "**PLEASE READ:** Before reporting here, *please* carefully "
+            "read our **Troubleshooting Guide**: <{0!s}> and search the issue "
+            "page for your error/problem, as most "
+            "posted bugs are duplicates or easy fixes.\n\n"
+            "Also, if you do post here, please describe step by step in "
+            "detail what you were doing when the error occurred. "
+            "Otherwise, your issue will likely be closed after 7 days. "
+            "Thanks.\n\n"
+            "**Note:** Delete this message before submitting."
+            ).format(__trouble_url__)
+        if traceback:
+            reminder_message = ("IMPORTANT: Read troubleshooting guide at "
+                                "{0!s} before posting!"
+                                ).format(__trouble_url_short__)
+        else:
+            traceback = "PASTE HERE"
         issue_template = """\
-## Description
+%s
 
-**What steps will reproduce the problem?**
+## What steps reproduce the problem?
 
 1. 
 2. 
 3. 
 
-**What is the expected output? What do you see instead?**
-
-
-**Please provide any additional information below**
-
+## Paste Traceback/Error Below
+(from `View > Panels > Internal Console`)
+```
 %s
+```
 
-## Version and main components
-
-* Spyder Version: %s %s
-* Python Version: %s
-* Qt Versions: %s, %s %s on %s
+## Versions
+Spyder %s %s
+Python %s
+Qt %s
+%s %s
+%s %s
 
 ## Dependencies
 ```
 %s
 ```
-""" % (traceback,
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800
+12345678001234567800123456780012345678001234567800123456780012345678001234567800

```

Comment:
What is this big block doing here? Is it an image or something?

Reply:
More questions:

* Is this the SHA1 string you were talking about above?
* Could you better explain why this is needed?
* Could you move it to the beginning of this file as a constant? It could be called `SHA1_REPORT_ERROR` or something like that.
* Since this constant seems to be the same string repeated several times, please generate it using the multiplication properties of strings in Python, i.e. `SHA1_REPORT_ERROR = initial_string * 20`.

Reply:
Its a total, rank, n00b mistake on my part; I must have somehow missed it despite usually being very careful to check ``git status`` and ``git diff`` every time before I commit. Its just a simple kludge for testing the maximum report length before cutoff that was never supposed to be committed, which happened despite several precautions I took to make sure I didn't screw up, as I often do. I'm sorry. Needless to say, it'll be gone in the next one.